backend/Application/Aggregates/Contract/Entities/ModuleReferenceEvent.cs

```diff
@@ -184,6 +187,21 @@ public class ModuleReferenceEventQuery
     [ExtendObjectType(typeof(ModuleReferenceEvent))]
     public sealed class ModuleReferenceEventExtensions
     {
+        /// <summary>
+        /// Returns module schema in a human interpretable form. Once present if the schema is embedded into the
+        /// Wasm module.
+        /// </summary>
+        public string? GetDisplaySchema([Parent] ModuleReferenceEvent module)
```

Comment:
This function doesn't seem to be used, and I don't understand (and can't find) what the `[Parent]` does? :pleading_face:

Reply:
Explained in person